### ISSUE TITLE
CodePackage activation error for no reason among other strange issues.

### ISSUE THREAD

**Issue by jfalameda** [microsoft/service-fabric#1056]
Service Fabric Runtime Version: **7.1.458.9590**

Environment: **Azure 5 nodes cluster**

I am having lots of strange issues regarding upgrading my application (see this related issue https://github.com/microsoft/service-fabric/issues/1054).

The issue I am facing now is that I am getting CodePackage activation errors on two services I am updating. I took a look at the ```D:\SvcFab\_App\``` app logs and I found several out and err files, which all of them are empty. Also, my startup code on those services reports to an external reporting service in case of exception on activation, which is not receiving any event (meaning the code is not even activating, thus the program is failing before starting).

This version is working without an issue on my stage and local environment (single-node clusters).

I am not sure what is going on, but it seems that some of the services are not even the ones supposedly deployed in the specified version (they seem to be older, as there is features mismatch for the exact same version deployed on stage).

I am scratching my head and trying everything possible as well as triple-checking my code while my whole production environment is on hold. I am under the impression that the cluster is severely broken, maybe related to the latest update.

OS: **Windows 10 Semiannual 2016 (Azure)**

**Comment by craftyhouse** [comment]
Please refer to this guidance for production issues https://github.com/microsoft/service-fabric/blob/master/CONTRIBUTING.md#open-an-issue
